Gems/AtomLyIntegration/ImguiAtom/Code/Source/DebugConsole.cpp

```diff
@@ -183,6 +165,25 @@ namespace AZ
         }
     }
 
+    void DebugConsole::AddDebugLog(const char* window, const char* debugLogString, AZ::LogLevel logLevel)
+    {
+        AZ::ILogger* logger = AZ::Interface<AZ::ILogger>::Get();
+        if (logger == nullptr || logLevel < logger->GetLogLevel())
+        {
+            return;
+        }
+
+        AZ::Color color = GetColorForLogLevel(logLevel);
+        if (strcmp(window, AZ::Debug::Trace::GetDefaultSystemWindow()) == 0)
+        {
+            AddDebugLog( debugLogString, color);
+        }
+        else
+        {
+            AddDebugLog( AZStd::string::format("(%s) - %s", window, debugLogString), color);
```

Comment:
nitpick extra space after left parenthesis
```suggestion
            AddDebugLog(debugLogString, color);
        }
        else
        {
            AddDebugLog(AZStd::string::format("(%s) - %s", window, debugLogString), color);
```